superbench/executor/executor.py

```diff
@@ -67,8 +68,7 @@ def __get_enabled_benchmarks(self):
 
     def __get_platform(self):
         """Detect runninng platform by environment."""
-        # TODO: check devices and env vars
-        return Platform.CUDA
+        return GPU().platform
```

Comment:
If we use the structure:
-devices
   --gpu
   --others
how __get_platform() to know devices? Here looks like a hardcode.

Reply:
updated